flink-libraries/flink-table/src/main/scala/org/apache/flink/table/functions/utils/TableSqlFunction.scala

```diff
@@ -53,22 +54,9 @@ class TableSqlFunction(
     paramTypes,
     functionImpl) {
 
-  /**
-    * Get the user-defined table function.
-    */
-  def getTableFunction = udtf
-
-  /**
-    * Get the type information of the table returned by the table function.
-    */
-  def getRowTypeInfo = rowTypeInfo
-
-  /**
-    * Get additional mapping information if the returned table type is a POJO
-    * (POJO types have no deterministic field order).
-    */
-  def getPojoFieldMapping = functionImpl.fieldIndexes
```

Comment:
Yes. POJOs are still supported. The pojo field mappings are re-calculated by the dynamic schema. You can see in `DataSetCorrelate` and `DataStreamCorrelate`. Also all the tests of POJOs still pass.